migrate-member-usernames.ts

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
Can this be in a migrations folder? / does it even make sense to have in this repo? Maybe rename it since backing up is only one part of it with this here.